cypress/integration/ete/delete.4.cy.ts

```diff
@@ -4,7 +4,7 @@ describe('Delete my account flow in Okta', () => {
 		cy.visit(
 			`/signin?returnUrl=${encodeURIComponent(
 				`https://${Cypress.env('BASE_URI')}/welcome/review`,
-			)}`,
+			)}&usePasswordSignIn=true`,
```

Comment:
does this mean we are not testing the default sign in method now ?

Reply:
We are! It's just in other test files that have already been added. Adding `usePasswordSignIn` to tests just means we don't have to update the authentication specific tests, e.g. for delete my account.